.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,73 @@
+name: Chromatic
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    paths:
+      - frontend-react/**
+    branches-ignore:
+      - deployment
+
+jobs:
+  chromatic-deployment:
+    name: Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find Comment
+        id: comment_find
+        uses: peter-evans/find-comment@v2
```

Comment:
We use [SHA checksums](https://devopsjournal.io/blog/2021/12/11/GitHub-Actions-Maturity-Levels) for all actions
`peter-evans/find-comment@034abe94d3191f9c89d870519735beae326f2bdb`

Reply:
Thanks for the callout!  I've updated -- feel free to take a look whenever!